devito/function.py

```diff
@@ -920,11 +929,13 @@ def coefficients(self):
     @property
     def point_symbols(self):
         """Symbol for coordinate value in each dimension of the point"""
-        return sympy.symbols('px, py, pz')
+        return tuple(sympy.symbols('p%s' % d) for d in self.grid.dimensions)
 
     @property
     def point_increments(self):
         """Index increments in each dimension for each point symbol"""
+        if self.grid.dim == 1:
```

Comment:
We should use this: https://docs.python.org/2/library/itertools.html#itertools.product